source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -923,7 +923,7 @@ class NameValuePairPropertyEditor extends PropertySheetEntry {
                 valueColumn.addItem(textInput);
 
                 let removeAction = new Adaptive.SubmitAction();
-                removeAction.iconUrl = require("svg-url-loader!../assets/xmark.svg");
+                removeAction.iconUrl = require("svg-url-loader!./assets/xmark.svg");
```

Comment:
@jwoo-msft Do you know of a good way to test this? The bug occurs when the designer package is consumed by another application.

Since we bootstrap our repo, I'm not sure how to test this here.